### PR TITLE
fix uihosting controller issue

### PR DESCRIPTION
**Pull request by simanerush**

### CODE REVIEW

File: CursedSnake/ViewsAndScenes/GameScene.swift [hunk 1]
```diff
@@ -223,7 +223,8 @@ class GameScene: SKScene, SKPhysicsContactDelegate {
         }
         else if touchedNode.name == "returnToMenu" {
             let vc = UIHostingController(rootView: TitleView())
-            view?.presentScene(vc)
+            // TODO: are there any edge cases when wither view or window are nil?
```
Comment (simanerush) [inline CursedSnake/ViewsAndScenes/GameScene.swift:226]: either*

Comment (aabagdi) [reply CursedSnake/ViewsAndScenes/GameScene.swift:226]: I don't think so? I'll take another look

Comment (aabagdi) [reply CursedSnake/ViewsAndScenes/GameScene.swift:226]: > either*

I don't think so? I'll take another look

Comment (simanerush) [reply CursedSnake/ViewsAndScenes/GameScene.swift:226]: ok then just leave the force unwraps and hope it doesn't crash

**Pull request merged**